raiden/network/proxies/utils.py

```diff
@@ -122,12 +122,15 @@ def log_transaction(log: BoundLoggerBase, description: str, details: Dict[Any, A
     token = uuid4()
     bound_log = log.bind(description=description, token=token, **details)
     try:
-        bound_log.debug("Entered")
+        bound_log.debug("Transation will be sent")
         yield
+    except RaidenRecoverableError:
+        bound_log.debug("Transaction invalidated", exc_info=True)
```

Comment:
This will print
```
Transaction invalidated
Transaction successful
```
which is confusing. Putting a `return` here would change that.

Reply:
hummm, I should have kept the raise